Makefile

```diff
@@ -285,16 +275,9 @@ download-key:
 	wget -O statping.gpg $(SIGN_URL)
 	gpg --import statping.gpg
 
-# push the :dev docker tag using curl
-dockerhub-dev:
-	docker build --build-arg VERSION=${VERSION} -t statping-ng/statping-ng:dev --no-cache -f Dockerfile.base .
-	docker push statping-ng/statping-ng:dev
-
```

Comment:
Note: (partially to myself because I _will_ forget this)
removing the push target for `:dev` while keeping the build target is a good thing since we don't want to push the dev tag but keep it for local testing (avoid confusion between a local-only tag and one that was pulled from somewhere)

you can just click "resolve conversation" on this comment once you've seen it ^^

Reply:
I mostly removed it because it used the base image which is weird. (Why push such a large image to Dockerhub? )